### PR TITLE
Initial draft of Nov 29 presentation

### PR DESCRIPTION
**Pull request by dcwalk**
Hey @dasanchez -- here is a first draft! So sorry for the delay, got slammed with work

Please push your changes directly to this! I'm gonna re-setup the repo to serve all our content so we should be able to use for the day :)

### CODE REVIEW

**Comment by dcwalk** [comment]
Hey Dante, all looks good, my only thought is that we should be consistent about the "outreach" v. "technical" projects, and I feel a little skittish but I'd avoid using the TPL logo until we go ahead with a workshop series, right now we are pre-proposal.

**Comment by dasanchez** [comment]
I like the changes you made!
- can you expand on the the outreach v technical consistency?
- agreed on the TPL logo, I'll take it out

File: presentations/20171129_tomesh-intro/index.html [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+					<br />
+					<br />
+					<br />
+					<br />
+					<br />
+				</section>
+
+				<section data-background="images/TPL-Logo.jpg" data-background-size="50%">
+					<p>
+						Multi-Week Workshops
```
Comment (dcwalk) [inline presentations/20171129_tomesh-intro/index.html]: Would cut this word and also the TPL logo

File: presentations/20171129_tomesh-intro/index.html [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+					<br />
+					<br />
+					<br />
+					<br />
+					<span>&nbsp;</span></p>
+				</section>
+
+				<section data-background="images/nodes.jpg" data-background-size="50%">
+					<p>
+						Install parties
```
Comment (dcwalk) [inline presentations/20171129_tomesh-intro/index.html]: Would cut this <p>

**Review by dcwalk — COMMENTED** [review]
Just the two slides!

**Comment by dasanchez** [comment]
Poof! Those two slides are now gone.

**Review by dasanchez — APPROVED** [review]

**Comment by dcwalk** [comment]
Okay, merging in so I can get this up?

**Pull request merged**